markdown.ipynb

```diff
@@ -4,21 +4,22 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `markdown.ipynb` Cell 22 Line 1_
```markdown
Here is an example of AppleScript:
```
Apple script review comment 3

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/24)._

Reply:
response to comment 3

Reply:
3